STITCH.md

```diff
@@ -0,0 +1,22 @@
+clump
+	stitch
+
+
+
+optimized for optinal fetches
+	optimized for non-optional fetches
+
+allows filtering
+	doesn't allow filtering
+
+multiple sourcing strategies
+	only positional sourcing
+
+caching
+	no caching
+
+simple execution model
+	query ast + execution plan + simplifications
+
+can batch nested flatmaps
```

Comment:
By the way I still don't have a full understanding of how this works :P, I'm hoping the [unit test](https://github.com/fwbrasil/clump/issues/36) will help me figure it out